autostart/qui-devices.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Icon=gtk-preferences
```

Comment:
Please, change the icon (here and in the actual widget) to something more related to devices. There are some in "devices" category in gnome-icon-theme package. If you have no better idea, use `drive-removable-media`.